src/utils/SpokeUtils.ts

```diff
@@ -14,6 +14,51 @@ import { toBytes32 } from "./AddressUtils";
 
 type BlockTag = providers.BlockTag;
 
+/**
+ * @param spokePool SpokePool Contract instance.
+ * @param deposit Deopsit instance.
+ * @param repaymentChainId Optional repaymentChainId (defaults to destinationChainId).
+ * @returns An Ethers UnsignedTransaction instance.
+ */
+export function populateRelay(
```

Comment:
Why'd you add this?

Reply:
Ah, I've had this kicking around for a while and Bennett fixed `populateV3Relay()` inbetween. I'll drop this.

Reply:
https://github.com/across-protocol/sdk/pull/906/commits/d1aaee98a31f08cf4da6e69cd6d4c280610f6732